packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -42,17 +71,50 @@ export const PanelChrome: React.FC<PanelChromeProps> = ({
   const headerStyles: CSSProperties = {
     height: headerHeight,
   };
-
+  const itemStyles: CSSProperties = {
+    minHeight: headerHeight,
+    minWidth: headerHeight,
+  };
   const containerStyles: CSSProperties = { width, height };
 
+  const handleMenuOpen = () => {};
+
+  const hasHeader = title || titleItems.length > 0 || menu;
+
   return (
     <div className={styles.container} style={containerStyles}>
-      <div className={styles.header} style={headerStyles}>
-        <div className={styles.headerTitle}>{title}</div>
-        {itemsRenderer(leftItems, (items) => {
-          return <div className={styles.leftItems}>{items}</div>;
-        })}
-      </div>
+      {hasHeader && !hoverHeader && (
+        <div className={styles.headerContainer} style={headerStyles} data-testid="header-container">
+          {title && <div className={styles.title}>{title}</div>}
+
+          {titleItems.length > 0 && (
+            <div className={styles.items} data-testid="title-items-container">
+              {titleItems.map((item, i) => (
+                <div key={`${item.icon}-${i}`} className={styles.item} style={itemStyles}>
+                  <IconButton tooltip={item.tooltip} name={item.icon} size="sm" onClick={item.onClick} />
+                </div>
+              ))}
+            </div>
+          )}
+
+          {menu && (
+            <div className={styles.item} style={itemStyles}>
+              <IconButton
+                tooltip={`Menu for ${title && `${title} `}panel`}
```

Comment:
Is this tooltip needed on the menu icon? I would expect the panel title to be shown in the tooltip on the title itself when ellipsis is applied.

Reply:
Yeah, not exactly needed - you're right. I meant it more as a `ariaLabel` which it should've been. I want it to be read by the screen reader, nothing else.